modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMMDFRequest.testcase

```diff
@@ -119,6 +119,89 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-172957. Verify that the review page displays all the information the partner user entered to the activity"
+	@priority = "5"
+	test ReviewPageDisplaysAllInputedInformation {
+		task ("Given I go to add a new MDF Request") {
+			PRMMarketing.gotoAddNewMDF();
+		}
+
+		task ("When filling the fields") {
+			var accountName = "Company Name 1";
+			var campaignName = "Campaign Name";
+			var countryName = "Global";
+			var goalsOptions = "Lead generation";
+			var targetMarketOptions = "Education";
+			var targetRolesOptions = "Administrator";
+			var activityDescription = "Activity Description";
+			var activityName = "Activity Name 1";
+			var budgetList = "Sponsorship Fee-2000";
+			var expense = "Sponsorship Fee";
+			var budget = "$2,000.00";
+			var leadList = "No";
+			var tactic = "Other";
+			var typeOfActivity = "Miscellaneous Marketing";
+			var startDate = DateUtil.getFormattedDate("+1", "M/dd/yyyy");
+			var endDate = DateUtil.getFormattedDate("+2", "M/dd/yyyy");
+		}
+
+		task ("Then you get the appropriate results") {
+			PRMMDFRequest.fillGoalsPage(
+				accountName = "${accountName}",
+				additionalOptions = "${additionalOptions}",
+				campaignDescription = "Campaign Description",
+				campaignName = "${campaignName}",
+				countryName = "${countryName}",
+				goalsOptions = "${goalsOptions}",
+				targetMarketOptions = "${targetMarketOptions}",
+				targetRolesOptions = "${targetRolesOptions}");
+
+			Button.click(button = "Continue");
+
+			PRMMDFRequest.newActivity(
+				activityDescription = "${activityDescription}",
+				activityName = "${activityName}",
+				budgetList = "${budgetList}",
+				followUpStrategy = "${followUpStrategy}",
+				leadFollowUpList = "${leadFollowUpList}",
+				leadList = "${leadList}",
+				tactic = "${tactic}",
+				targetOfLeads = "${targetOfLeads}",
+				typeOfActivity = "${typeOfActivity}");
+
+			Button.click(button = "Continue");
+
+			Button.click(button = "Continue");
+		}
+
+		task ("And verify that all the filled in fields are present") {
```

Comment:
Hi @marinaasant0s, could you please update this task to: "Then all inputted information are displayed on Review Page"